src/Localization/.vsconfig

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
@CodeBlanch do you know what this is for?

Reply:
TBH, no clue! I copied from [/src/Middleware/.vsconfig](https://github.com/dotnet/aspnetcore/blob/master/src/Middleware/.vsconfig). There were 4 files in the root there that I copied and then modified for the Localization tree. Looking around at some of the other folders, it might even be missing one (build.sh)?

Reply:
@Pilchie, are these vsconfig files useful with slnf?

Reply:
> What is it for?

They put a warning in the top of the Solution Explorer if your install doesn't have all of the components listed, and allow you to easily select those components for addition in the setup UI.

> are these vsconfig files useful with slnf?

I don't *think* so.  I'm pretty sure they only work if they are directly beside the actual .sln file.